autoloader.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import os, sys, time
+
+def files_to_timestamp(path):
+    files = [os.path.join(path, f) for f in os.listdir(path)]
+    return dict ([(f, os.path.getmtime(f)) for f in files])
+
+if __name__ == "__main__":
+
+    path_to_watch = "/home/jyotirmoy/Documents/gms-artic/data/"
+    #sys.argv[1]
+    print('Watching {}..'.format(path_to_watch))
+
+    before = files_to_timestamp(path_to_watch)
+
+    while 1:
+        time.sleep (2)
```

Comment:
incredibly frequent loop

Reply:
the script is only required to run in the background to check auto-uploading of data files. I don't think this is required to run the pipeline.